src/main/java/com/raisetech/controller/BookController.java

```diff
@@ -0,0 +1,26 @@
+package com.raisetech.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import com.raisetech.entity.Book;
+import com.raisetech.mapper.BookMapper;
+
+@Controller
+public class BookController {
+	
+	@Autowired
+	BookMapper bookMapper;

```

Comment:
こちらはコンストラクタインジェクションと言う手法があるため、そちらを使っていきましょうか！
また、なぜコンストラクタインジェクションが推奨されるかもぜひお調べください！
```suggestion
	BookMapper bookMapper;

    @Autowired
    public MachinesController(BookMapper bookMapper) {
        this.bookMapper = bookMapper;
    }
```

Reply:
@YudaiNoda3576 
こちら私も同意です！